src/ui/src/js/controllers/request_view.js

```diff
@@ -33,6 +33,33 @@ export default function requestViewController(
     request) {
   $scope.service = RequestService;
 
+ $scope.download = function() {
+      var filename = $scope.data.id;
+      var output;
+      var outputType = $scope.data.output_type;
```

Comment:
It's a *lot* easier to declare variables with `let` instead of `var`. The rules regarding when and where `var` variables exist are really complicated and not worth it. Trust me, just use `let` everywhere :smile: